docs/dev/query-type-conversion.md

```diff
@@ -74,17 +74,18 @@ Runtime:
  Evaluation result: *false*
 ```
 
-Therefore type precedence is supposed to be defined based on semantic expected rather than intuitive “width” of type. Now let’s reverse the direction and make Boolean the parent of String type.
+Therefore type precedence is supposed to be defined based on semantic expected rather than intuitive 'width' of type. Now let's reverse the direction and make Boolean the parent of String type.
 
 ![New type hierarchy](img/type-hierarchy-tree-with-implicit-cast.png)
+Note: type hierarchy structure shown on the picture below was implemented in [#166](https://github.com/opensearch-project/sql/pull/166), but was changed later.
 
 ```
 Compiling time:
  Expression: false = 'FALSE'
  Unresolved signature: equal(BOOL, STRING)
  Resovled signature: equal(BOOL, BOOL)
  Function builder: 1) returns equal(BOOL, cast_to_bool(STRING)) impl
-                   2) returns equal(BOOL, BOOL) impl
+                   1) returns equal(BOOL, BOOL) impl
```

Comment:
Should this not be a 2) considering the line above is 1)?

Reply:
Fixed in b04a92e1f